src/main/java/com/project/food_ordering_service/domain/delivery/exception/DeliveryNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.project.food_ordering_service.domain.delivery.exception;
+
+public class DeliveryNotFoundException extends RuntimeException {
+    public DeliveryNotFoundException() {
+        super();
+    }
+}
```

Comment:
```suggestions 
@ResponseStatus(code = HttpStatus.NOT_FOUND)
public class DeliveryNotFoundException extends RuntimeException {
    public DeliveryNotFoundException() {
        super();
    }
}

```
이런식으로 위에 상태코드를 먼저 적어주면 좋을것 같습니다
@mistyblue0302